pkgs/by-name/au/automatic-ripping-machine/package.nix

```
@@ -0,0 +1,44 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname   = "automatic-ripping-machine";
+  version = "2.17.4";
+
+  src = fetchFromGitHub {
+    owner  = "automatic-ripping-machine";
+    repo   = "automatic-ripping-machine";
+    tag    = "${finalAttrs.version}";
+    sha256 = "sha256-1EoRiNQUvV/PdI77n+vkE0tSHUP6NtidpCcfdwI2tCc=";
+  };
+
+  # No compilation needed; just copy the source files
+  buildPhase = "true";
```

```suggestion
  dontBuild = true;
```

https://nixos.org/manual/nixpkgs/unstable/#var-stdenv-dontBuild